interface/patient_file/encounter/load_form.php

```diff
@@ -21,6 +21,11 @@
 if (substr($_GET["formname"], 0, 3) === 'LBF') {
```

Comment:
@juggernautsei do you have thoughts on making this an event instead of hard-coding the directory?  This is short and simple but not as flexible, just wondering your thoughts on this?  I need to actually deal with loading forms in the modules as well and my preference is using events but I'd like to know your thoughts on this one over an event piece?

Reply:
Stephen, I appreciate you asking me about this. I did this as a precursor to making it what you are suggesting an event. This is working and I was trying to be thoughtful of using the standard nomenclature the modules were started with even though you can call a module anything and it will work now. But it would not take to much to add and event here and anyone listening can act. I can work on adding that event. I have some work around this that will not take me far out of the way. Should I leave this open or close it?

Reply:
So you bypass the form registry all together?

Reply:
yes

Reply:
@juggernautsei How are you handling the report.php and the view.php pieces that come into play with forms.php and view_form.php in the encounter display?  I'm moving a custom form for a customer out of core into their module and it looks like those areas have to be touched as well.